acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -110,41 +111,51 @@ public void thenItIsVisibleToConsumers() throws Throwable {
         assertThat("Publication summary is as expected",consumablePublicationPage.getSummaryText(),
             is(publication.getPublicationSummary()));
 
-        assertThat("Uploaded attachment is available",consumablePublicationPage.getAttachmentName(),
-            is(publication.getAttachment().getName()));
+        assertThat("Geographic coverage is as expected",consumablePublicationPage.getGeographicCoverage(),
+            is("Geographic coverage:\n" + publication.getGeographicCoverage()));
 
-        assertThat("Uploaded attachment's size is displayed",consumablePublicationPage.getAttachmentSizeText(),
-            is("size: " + publication.getAttachment().getContent().length + " B."));
+        assertThat("Publication information type is as expected",consumablePublicationPage.getInformationType(),
+            is("Information types:\n" + publication.getInformationType()));
 
-        if (acceptanceTestProperties.isHeadlessMode()) {
-            // At the moment of writing, there doesn't seem to be any easy way available to force Chromedriver
-            // to download files when operating in headless mode. It appears that some functionality has been
-            // added to DevTools but it's not obvious how to trigger that from Java so, for now at least,
-            // we'll only be testing file download when operating in a full, graphical mode.
-            //
-            // See bug report at https://bugs.chromium.org/p/chromium/issues/detail?id=696481 and other reports
-            // available online.
-            log.warn("Not testing file download due to running in a headless mode.");
-        } else {
+        assertThat("Expected number of attachments is displayed.",
+            consumablePublicationPage.getAttachments().size(),
+            is(publication.getAttachments().size()));
 
-            // Download the file - simulate mouse click on the <a> tag.
-            consumablePublicationPage.getAttachmentElement().click();
+        publication.getAttachments().forEach(attachment -> {
```

Comment:
Is it worth breaking some of this out into separate methods to avoid this one being so long?

Reply:
Done.